scram-tools.file/tools/photospp/photospp.xml

```diff
@@ -1,11 +1,13 @@
 <tool name="photospp" version="@TOOL_VERSION@" revision="1">
```

Comment:
@mkirsano , as toolfile definition is changed so please please increment the revision here e.g. change it to `revision="2"`